src/go/configgenerator/listener_generator.go

```diff
@@ -601,11 +633,21 @@ func makeTranscoderFilter(serviceInfo *sc.ServiceInfo) *hcmpb.HttpFilter {
 				DescriptorSet: &transcoderpb.GrpcJsonTranscoder_ProtoDescriptorBin{
 					ProtoDescriptorBin: configContent,
 				},
-				IgnoredQueryParameters: []string{"api_key", "key", "access_token"},
-				AutoMapping:            true,
-				ConvertGrpcStatus:      true,
+				AutoMapping:       true,
+				ConvertGrpcStatus: true,
+			}
+			for apiKeyQueryParam := range serviceInfo.ApiKeyQueryParams {
+				transcodeConfig.IgnoredQueryParameters = append(transcodeConfig.IgnoredQueryParameters, apiKeyQueryParam)
+			}
+
+			for jwtQueryParam := range serviceInfo.JwtQueryParams {
+				transcodeConfig.IgnoredQueryParameters = append(transcodeConfig.IgnoredQueryParameters, jwtQueryParam)
 			}
+
+			sort.Sort(sort.StringSlice(transcodeConfig.IgnoredQueryParameters))
```

Comment:
Any specific reason for the sort? Is it just for readability?

Reply:
Just curious, still have the same question

Reply:
Stable order for test